service/src/main/resources/application.properties

```diff
@@ -62,3 +62,5 @@ elasticsearch.scheme=http
 livy.uri=http://localhost:8998/batches
 # yarn url
 yarn.uri=http://localhost:8088
+# griffin event listener
+#internal.event.listeners=org.apache.griffin.core.event.JobEventHook
```

Comment:
this example uses outdated format, right?

Reply:
correct, change it as definition in test folder